bin/improver-ecc

```diff
@@ -0,0 +1,154 @@
+             'percentiles based on the order of the '
+             'raw ensemble forecast.')
+    group.add_argument('--rebadging', default=False, action='store_true',
+        help='The option used to create ensemble members '
+              'from percentiles by rebadging the input '
+              'percentiles.')
+
+    # If reordering, can do so either based on original members, or randomly.
+    reordering = parser.add_argument_group(
+        'Reordering options', 'Options for reordering the input percentiles '
+        'using the raw ensemble forecast as required to create ensemble '
+        'members.')
+    reordering.add_argument('--raw_forecast_filepath',
+                        metavar='RAW_FORECAST_FILE',
+                        help='A path to an raw forecast NetCDF file to be '
+                             'processed.')
+    reordering.add_argument('--random_ordering', default=False,
+                        action='store_true',
+                        help='Decide whether or not to use random ordering '
+                             'within the ensemble reordering step.')
+    reordering.add_argument('--fixed_random_seed', default=None,
+                        help='Decide whether or not a fixed random seed is '
+                             'used to create the random numbers, either '
+                             'used for the random_ordering or for splitting '
+                             'tied values.')
+
+    rebadging = parser.add_argument_group(
+        'Rebadging options', 'Options for rebadging the input percentiles '
+        'as ensemble members.')
+    rebadging.add_argument('--member_numbers', default=None,
+        metavar='MEMBER_NUMBERS',
+        help='A list of ensemble member numbers to use '
+              'when rebadging the percentiles into members.')
+
+    args = parser.parse_args()
+
+    # CLI argument checking:
+    # Can only do one of reordering or rebadging: if options are passed which
+    # correspond to the opposite method, raise an exception.
+    # Note: Shouldn't need to check that both/none are set, since they are
+    # defined as mandatory, but mutually exclusive, options.
+    if args.reordering:
+        if np.any([args.member_numbers]):
+            parser.wrong_args_error('member_numbers', 'reordering')
+    elif args.rebadging:
+        if np.any([args.raw_forecast_filepath, args.random_ordering]):
+            parser.wrong_args_error(
+                'raw_forecast_filepath, random_ordering', 'rebadging')
+
+
+    # Safe to now actually do the work...
+    cube = iris.load_cube(args.input_filepath)
+
+    # TODO: checking should eventually be moved into the framework itself:
+    if not cube.coords('percentile_over_realization'):
+        msg = ('The input cube: {} does not have a '
+               'percentile_over_realization dimension.'.format(cube))
+        raise CoordinateNotFoundError(msg)
+
+    # For now, assume only doing percentiles -> percentiles
```

Comment:
Also should probably add a comment to say this does nothing if number of output percentiles = number of input percentiles.  And/or in the help information